packages/display/src/DisplayObject.ts

```diff
@@ -18,7 +18,194 @@ export interface DisplayObject extends GlobalMixins.DisplayObject, EventEmitter
 /**
  * The base class for all objects that are rendered on the screen.
  *
- * This is an abstract class and should not be used on its own; rather it should b e extended.
+ * This is an abstract class and can not be used on its own; rather it should be extended.
+ *
+ * ## Display objects implemented in PixiJS
+ *
+ * | Display Object                  | Description                                                           |
+ * | ------------------------------- | --------------------------------------------------------------------- |
+ * | {@link PIXI.Container}          | Adds support for `children` to DisplayObject                          |
+ * | {@link PIXI.Graphics}           | Shape-drawing display object similar to the Canvas API                |
+ * | {@link PIXI.Sprite}             | Draws textures (i.e. images)                                          |
+ * | {@link PIXI.Text}               | Draws text using the Canvas API internally                            |
+ * | {@link PIXI.BitmapText}         | More scaleable solution for text rendering, reusing glyph textures    |
+ * | {@link PIXI.TilingSprite}       | Draws textures/images in a tiled fashion                              |
+ * | {@link PIXI.AnimatedSprite}     | Draws an animation of multiple images                                 |
+ * | {@link PIXI.Mesh}               | Provides a lower-level API for drawing meshes with custom data        |
+ * | {@link PIXI.NineSlicePlane}     | Mesh-related                                                          |
+ * | {@link PIXI.SimpleMesh}         | v4-compatibile mesh                                                   |
+ * | {@link PIXI.SimplePlane}        | Mesh-related                                                          |
+ * | {@link PIXI.SimpleRope}         | Mesh-related                                                          |
+ *
+ * ## External libraries that export or service display objects
+ *
+ * | Package                         | Description                                                          |
+ * | ------------------------------- | -------------------------------------------------------------------- |
+ * | pixi-layers                     | Faster layering of display objects than z-indexing                   |
+ * | pixi-sdf-text                   | Signed distance field text implementation                            |
+ * | @pixi-essentials/transformer    | Interactive interface for dragging, scaling, rotating display objects|
+ *
+ * ## Transforms
+ *
+ * The [transform]{@link DisplayObject#transform} of a display object describes the projection from its
+ * local coordinate space to its parent's local coordinate space. The following properties are derived
+ * from the transform:
+ *
+ * <table>
```

Comment:
Maintaining more complicated HTML formatting in these comments is kind of annoying to deal with. Would be in favor of using sub-headers (h4, h5, etc) and doing these inline instead. A table doesn't add much her imo.

Reply:
@bigtimebuddy I would be in favor of using tables here. It gives the user of a sense of "one block" and "if I know it, I can skip it". I've optimized it for the generated docs, not for our readability 🤔

Reply:
My point is that it's harder to maintain than markdown and easy to mess up. But I'm not bullish on it if you think it's a better user experience ultimately.

Reply:
@bigtimebuddy I understand that Markdown is generally better for tables. But this table is very large, and then you can't create paragraphs in Markdown tables. So Markdown would've made it hard to create a table with more than 200 characters per cell.